packages/testssl.sh/fix-hardcodepath.patch

```diff
@@ -0,0 +1,96 @@
+--- a/testssl.sh
++++ b/testssl.sh
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env bash
++#!@TERMUX_PREFIX@/bin/env bash
+ #
+ # vim:ts=5:sw=5:expandtab
+ # we have a spaces softtab, that ensures readability with other editors too
+@@ -99,11 +99,11 @@
+ if [[ "$SHELLOPTS" =~ xtrace ]]; then
+      if "$DEBUGTIME"; then
+           # separate debugging, doesn't mess up the screen, $DEBUGTIME determines whether we also do performance analysis
+-          exec 42>&2 2> >(tee /tmp/testssl-$$.log | sed -u 's/^.*$/now/' | date -f - +%s.%N >/tmp/testssl-$$.time)
++          exec 42>&2 2> >(tee @TERMUX_PREFIX@/tmp/testssl-$$.log | sed -u 's/^.*$/now/' | date -f - +%s.%N >/tmp/testssl-$$.time)
```

Comment:
@ifurther The first part is fixed. What's up with second (`>/tmp/testssl-$$.time`)? There also numerous places for where other paths are not fixed.